brave/.SRCINFO

```diff
@@ -1,7 +1,7 @@
 pkgbase = brave
 	pkgdesc = A web browser that stops ads and trackers by default
-	pkgver = 1.46.140
-	pkgrel = 1
+	pkgver = 1.46.144
+	pkgrel = 2
```

Comment:
No big deal I'll fix this, bug the `pkgrel` should always be relative to what is posted on the AUR, and always reset to 1 when the `pkgver` changes. I have to do local builds too with all sorts of pkgrel bumps (I often use dot notation like `1.1` for unpublished local test builds), but the repos should only be bumped if they need bumping over an AUR published version.